nexus/0.8.1/nexus-operator.v0.8.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,169 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
the version named directory needs to include the product name.

i.e. rename directory `0.8.1` to `nexus-0.8.1`